noteable/sql/connection.py

```diff
@@ -133,29 +131,7 @@ def factory_and_register(
 
         If we encounter an exception at factory time, then remember it as a bootstrapping failure.
         """
-        try:
-            conn = self.factory(sql_cell_handle, human_name, connection_url, **kwargs)
-        except Exception as e:
-            # Eat any exceptions coming up from trying to describe the connection down into SQLAlchemy.
-            # Bad data entered about the datasource that SQLA hates?
-            #
-            # If we don't eat this, then it will ultimately break us before we make the call to register
-            # the SQL Magics entirely, and will get errors like '%%sql magic unknown', which is far
-            # worse than attempts to use a broken datasource being met with it being unknown, but other
-            # datasources working fine.
-            logger.exception(
-                'Unable to bootstrap datasource',
-                sql_cell_handle=sql_cell_handle,
-                human_name=human_name,
-                exception=str(e),
-            )
-
-            # Remember the failure so can be shown if / when human tries to use the connection.
-            self.add_bootstrapping_failure(
-                sql_cell_handle=sql_cell_handle, human_name=human_name, error_message=str(e)
-            )
-
-            return
+        conn = self.factory(sql_cell_handle, human_name, connection_url, **kwargs)
```

Comment:
Adding to the code in the try/except block in the bootrstapping call then simplified the factory() call not needing to try / catch.